### PR TITLE
community: add new committer

### LINKED ISSUE #42: [Proposal] New committer proposal
As [`CONTRIBUTORS`](https://github.com/X-lab2017/github-analysis-report-2020/blob/master/CONTRIBUTORS) file shows, @sunshinemingo is quite active for past two weeks in project discussion, I propose @sunshinemingo to be our new committer to approve PR.

### PR DESCRIPTION
**Pull request by frank-zsy**
Signed-off-by: Frankzhaopku <syzhao1988@126.com>

Add @sunshinemingo to committer.

Close #42

### CODE REVIEW

**Comment by frank-zsy** [comment]
I need another committer to approve this~ @will-ww @WuShaoling @heming6666

**Review by heming6666 — APPROVED** [review]

**Comment by frank-zsy** [comment]
@heming6666 we need `approve` command to do this, or maybe we should support GitHub approve event to add the label.

**Comment by heming6666** [comment]
> We need `approve` command to do this.

Yeah, I knew that. And I just want to show my own opinion on the addition of commiter. We can hear opinions from other guys before merging this PR.  

> Maybe we should support GitHub approve event to add the label.

In this case, we don't need support this. But if someone doesn't like type in `/approve` command, we can support this and add configuration in `hypertrons.json`, like:

```
{
    // Auto add pull/approved label when receiving approve event
    'auto-merge': true,
}
```

**Pull request merged**
